src/scripts/onboard.ts

```
@@ -0,0 +1,73 @@
+import { join } from "path";
+process.env.NODE_CONFIG_DIR = join(__dirname, "..", "config");
+import { FileUtils, Logger, UserUtils } from "../utils";
+
+/**
+ * Get the base URL for the Azure DevOps organization
+ *
+ * @returns {Promise<string>} Base URL
+ */
+async function getBaseUrl(): Promise<string> {
+  return UserUtils.askQuestionWithDefault(
+    "What is your Azure DevOps test instance base url?",
```

I might be misunderstanding the use case here, but from a user perspective, I'm not sure if we should be asking this with a default value. If I'm clicking through and not paying attention and just hitting enter with this default value, I'd have to rerun the script anyway. I'd rather just throw an error or continue asking until they give the url they want to use.

---

Good point. I'll remove the default